### DISCUSSION TITLE
Animate toggleded content

### DISCUSSION THREAD

**Discussion by simonferndriger** [alpinejs/alpine#3517] [1. Help] [answered]
When using `x-show` you have the option to smooth the appearance of the toggled (inline) content a bit using `x-transition`. However, the content _below_ the newly displayed element jumps down without transition. Is it possible to smooth this content jumping (similar to `jQuery.slideDown()`) as well somehow?

**Comment by ekwoka** [comment, marked as answer]
Yes: https://alpinejs.dev/plugins/collapse
